db/migrate/20220712221741_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.string :timestamps
+      t.references :users, null: false, foreign_key: true
```

Comment:
- [ ] **Required:** Following the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the user table's foreign key has been given an alias of `author_id`, kindly implement this throughout all models that `belongs_to`  to the user. See this article for [more info.](https://medium.com/@pawlkris/activerecord-naming-conventions-and-how-to-alias-around-them-afd6b357d479)

Reply:
Thanks, @brytebee I saw the document you referenced but it is not enough to resolve the alias of that reference. But as you suggest that this is the correct way to resolve the alias, I going to explore more about how to solve this issue.

Reply:
I resolved the alias using the Add a reference column to a table command.